content/en/network_monitoring/devices/guide/device_profiles.md

```diff
@@ -23,61 +23,130 @@ further_reading:
+### Step 2: Profile inheritance 
 
-### Global Tags
+Use profile inheritance to adopt configurations such as metadata, metrics, and tags. This simplifies scaling your device profiles and allows you to build on existing ones. Datadog automatically includes some inherited profiles, (`_base.yaml` `_generic-if.yaml`, `_generic-ip.yaml`, `_generic-ospf.yaml`, `_generic-tcp.yaml` and `_generic-udp.yaml`), which are recommended **not** to be removed. 
 
-Add global tags for more advanced and granular options, which allows you to assign a weight to a specific metric.
+Reference the [Supported Device Profiles][16] for the full list of inherited profiles.
 
-{{< img src="/network_device_monitoring/profile_onboarding/Add_global_tags.png" alt="The Network Device profile creation page showing the Add Global Tags dropdown" style="width:100%;">}}
+1. Keep the Datadog `_base.yaml` profile, and any other inherited Datadog profiles specific to your needs. Optionally, you can select additional profiles to inherit. The relevant fields appear on the right under Inherited Profiles, with an `Inherited` tag next to any inherited metrics, tags, or metadata:
 
-#### Advanced Options
+   {{< img src="/network_device_monitoring/profile_onboarding/profile_inheritance.png" alt="The Network Device profile creation page showing the Profile inheritance section." style="width:100%;">}}
 
-| Modification       | Description   |
-| ------------- | ------------- |
-| No Modification | The device's returned value will be used directly as the tag value. |
-| Format | This can be [mac_address][5] or [ip_address][6]. |
-| Extract Value | A regular expression used to [extract][7] the tag value from the SNMP value provided by the device. |
-| Mapping | This is described [here][8]. |
+    **Note**: Changes made to the parent profile are automatically applied to the child profiles. If you need to adjust any inherited attributes in the child profiles, such as metrics, tags, or metadata, you must modify the parent profile.
 
+### Step 3: Select reference devices
 
-### Scalar metrics
+Use reference devices to select which devices you want to gather OIDs for your chosen device models. The **reference devices** field is pre-selected based on the `SysObjectID` that was specified in the [profile details](#step-1-profile-details).
 
-See [Metrics Advanced Options](#metrics-advanced-options).
+1. Keep the current reference device selection to perform a device scan. Additionally, you can add more devices or change the current selection.
 
-### Tabular metrics
+2. Optionally, click **Proceed Manually** to proceed without performing a scan.
 
-See [Metrics Advanced Options](#metrics-advanced-options).
+  {{< img src="/network_device_monitoring/profile_onboarding/reference_devices.png" alt="The Network Device profile creation page showing the Reference device section." style="width:100%;">}}
 
-### Add Tabular Tags
+2. Click **Scan Devices** to proceed to Step 4, which initiates the scan.
 
```

Comment:
I would move the primary instruction above the image, i almost missed it when reading